src/components/collect-qr.vue

```diff
@@ -36,19 +35,9 @@ export default {
     }
   },
   computed: {
-    fullSettings() {
-      return {
-        general: {
-          ...this.settings,
-          server_url: `${window.location.origin}${this.settings.server_url}`
-        },
-        // Collect requires the settings to have an `admin` property.
-        admin: {}
-      };
-    },
```

Comment:
`FieldKeyQrPanel` and `FormDraftTesting` now do this work. I thought about leaving it here, but the code would have been surprisingly hairy, since `settings` now contains nested objects. I also think the new approach is a little clearer. Part of me doesn't love that `FieldKeyQrPanel` and `FormDraftTesting` now have some duplicate code, but it's just two components and not a lot of duplicate code. And probably we'll return to all this before too long once we start working on project settings.